omnibus/config/software/datadog-logs-agent-win.rb

```diff
@@ -10,17 +10,17 @@
 
 default_version "db/windows_svc"
 
-source git: 'https://github.com/DataDog/datadog-log-agent.git'
-relative_path 'src/github.com/DataDog/datadog-log-agent'
+source git: 'https://github.com/DataDog/datadog-agent.git'
+relative_path 'src/github.com/DataDog/datadog-agent'
```

Comment:
is this required, now that we're in same repository? Or do we not care yet?

Reply:
We don't care yet but we might need this in the future when we'll support windows